create-hpc-cluster-linux-cn/azuredeploy.json

```diff
@@ -159,400 +159,122 @@
     }
   },
   "variables": {
-    "apiVersion": "2015-05-01-preview",
-    "location": "[resourceGroup().location]",
     "virtualNetworkName": "[concat(parameters('clusterName'), 'VNet')]",
     "storageAccountNamePrefix": "[toLower(concat('hpc', uniqueString(resourceGroup().id, parameters('clusterName'))))]",
```

Comment:
You'll need to limit the clusterName parameter 8 characters otherwise the concat string will be more than 24 chars of the allowed storage account name. The other option is to do the below

```[concat('hpccluster', uniqueString(resourceGroup().id))]```